stage0/src/lib.rs

```diff
@@ -43,8 +42,8 @@ pub mod paging;
 mod sev;
 mod zero_page;
 
-#[global_allocator]
-static BOOT_ALLOC: Bump<[u8; 128 * 1024]> = Bump::uninit();
+// Reserve 128K for boot data structures.
```

Comment:
Pre-existing: We know 128K will be enough?

Reply:
No allocations have failed yet :)

In practice, we use <10 4K structures (page tables, zero page, etc) + some bytes for the kernel cmdline, so well below 128K.